nss-day-cohort-39/nutshell-smaragdine-sugar-gliders
# Added Friends List Component

**Changes**
- Added logic to render the friends list component to the UI

**Testing**
- Serve the application to localhost
- Register as a user or sign in with the following test info (username: Steve Brownlee Password: coach)
- Ensure that the friends list component appears in the dashboard. 
- Click the Find Friend button and add a friend
- Click the Remove Friend button to remove a friend from the list
- Ensure that all other components are still functioning properly.
